nvtabular/ops/add_metadata.py

```diff
@@ -36,6 +34,42 @@ def transform(self, col_selector: ColumnSelector, df: DataFrameType) -> DataFram
     def output_tags(self):
         return self.tags
 
+
+class AddMetadata(AddTags):
+    """
+    This operator will add user defined tags and properties
+    to a Schema.
+    """
+
+    def __init__(self, tags=None, properties=None):
+        super().__init__(tags)
+        self.properties = properties or {}
+
     @property
     def output_properties(self):
         return self.properties
+
+
+# Wrappers for common features
```

Comment:
These classes look good. 👍🏻 

They should use the canonical tags (`Tags.USER` for user features, etc) from `merlin-core` though: https://github.com/NVIDIA-Merlin/core/blob/main/merlin/schema/tags.py#L20